SAP/spartacus
# chore: Update customer ticket e2es

closes https://jira.tools.sap/browse/CXSPA-2278

to test: 

1. remove the .skip from the describe on all 3 tests
2. make sure you're running against the 'new' server https://20.127.226.23:9002, which contains the sample data

- [ ] to verify - repeatability
